terraform/main.tf

```diff
@@ -518,5 +517,5 @@ output "use_test_pha_decryption_key" {
 }
 
 output "has_test_environment" {
-  value = lookup(var.test_peer_environment, "env_with_ingestor", "") == var.environment
+  value = length(module.fake_server_resources) != 0
```

Comment:
So it seems if you don't somehow use the output of a targeted module, these never get updated.